src/Security/Security.php

```diff
@@ -706,15 +929,30 @@ public function createCredentials($strategy, $kuid, $credentials, array $options
      * @param $strategy
      * @param $kuid
      * @param array $options
+     *
      * @return mixed
+     *
+     * @throws InvalidArgumentException
      */
     public function deleteCredentials($strategy, $kuid, array $options = [])
     {
-        $options['httpParams'][':strategy'] = $strategy;
-        $options['httpParams'][':kuid'] = $kuid;
+        if (empty($strategy) || empty($kuid)) {
+            throw new InvalidArgumentException('Kuzzle\Security::deleteCredentials: strategy and kuid are required');
+        }
 
+        $options['httpParams'] = [
+            ':strategy' => $strategy,
+            ':kuid' => $kuid
+        ];
 
-        return $this->kuzzle->query($this->buildQueryArgs('deleteCredentials'), [], $options)['result'];
+        return $this->kuzzle->query(
+            $this->kuzzle->buildQueryArgs('security', 'deleteCredentials'),
+            [
```

Comment:
This body is to be removed: params come from the options since we're performing a HTTP DELETE